pyuvdata/tests/test_uvbase.py

```diff
@@ -230,3 +233,10 @@ def test_location_degree_none():
 def test_warning():
     output = _warning("hello world")
     assert output, "hello world\n"
+
+
+def test_check_acceptability_only():
```

Comment:
This test name confuses me, it's testing the `only_check_set` option, not the `run_check_acceptability` option.